examples/trusted_information_retrieval/proto/trusted_information_retrieval.proto

```diff
@@ -42,3 +42,41 @@ service TrustedInformationRetrieval {
   // Find the nearest Point Of Interest based on linear search in the database.
   rpc ListPointsOfInterest(ListPointsOfInterestRequest) returns (ListPointsOfInterestResponse);
 }
+
+message DatabaseEntry {
+  uint32 id = 1;
+  string name = 2;
+  string terminal_name = 3;
+  float latitude_degrees = 4;
+  float longitude_degrees = 5;
+  bool installed = 6;
+  bool locked = 7;
+  string install_date = 8;
+  string removal_date = 9;
+  bool temporary = 10;
+  uint32 number_of_bikes = 11;
+  uint32 number_of_empty_docks = 12;
+  uint32 number_of_docks = 13;
+}
+
+message GetDatabaseSizeRequest {}
+
+message GetDatabaseSizeResponse {
+  int32 size = 1;
+}
+
+message ListDatabaseEntriesRequest {
+  int32 first_entry = 1;
+  int32 last_entry = 2;
```

Comment:
Same comment as above re: `first` and `page_size` https://cloud.google.com/apis/design/design_patterns#list_pagination (no need to mess around with pagination tokens for now, it would just complicate things)

Reply:
Pagination tokens will be necessary for stopping requests due to the end of the database.

Reply:
Done